wren-ui/src/components/diagram/index.tsx

```diff
@@ -123,6 +137,8 @@ const ReactFlowDiagram = forwardRef(function ReactFlowDiagram(
           nodeTypes={nodeTypes}
           edgeTypes={edgeTypes}
           maxZoom={1}
+          onPaneClick={(event) => dispatchMouseEvent(event)}
```

Comment:
I think `onPaneClick` can be removed since `onPointerDown` is executed in every pointer in Reactflow component

Reply:
Ok, I'll look into this right now